android/engine/src/main/java/org/smartregister/fhircore/engine/data/local/patient/PatientRegisterRepository.kt

```diff
@@ -60,6 +60,6 @@ constructor(
   ): ProfileData? =
     registerDaoFactory.registerDaoMap[healthModule]?.loadProfileData(
       appFeatureName = appFeatureName,
-      patientId = patientId
+      resourceId = patientId
```

Comment:
Any reason why this parameter was renamed to `resourceId`?

Reply:
The mentioned Id could be a patientId or groupId,
hence it's renamed to resourceId.